ietf-layer0-types-ext.yang

```diff
@@ -613,17 +687,17 @@ module ietf-layer0-types-ext {
       and associated penalty";
 
     leaf max-polarization-dependent-loss {
-      type decimal64 {
-        fraction-digits 2;
-      }
-      units "dB";
+      type power-in-db-or-null;
       config false;
       mandatory true;
       description
         "Maximum acceptable accumulate polarization dependent loss";
     }
     leaf penalty {
-      type uint8;
+      type union {
+        type uint8;

```

Comment:
Shouldn't penalty be a decimal value ?

Reply:
To be discussed with other co-authors in the upcoming weekly call (after the I-D submission)

Co-Authored-By: sergio belotti <sergio.belotti@nokia.com>